lib/active_record/connection_adapters/sqlserver/jdbc.rb

```diff
@@ -0,0 +1,49 @@
+begin
+  eval 'com.microsoft.sqlserver.jdbc.SQLServerDriver'
+rescue NameError
+  begin
+    require 'sqljdbc4'
+  rescue
+    raise("com.microsoft.sqlserver.jdbc.SQLServerDriver not loaded, try installing the sqljdbc4 gem or put the jdbc driver jar in the java CLASSPATH")
+  end
+end
+
+module ActiveRecord
+  module ConnectionAdapters
+    module SQLServer
+      module Jdbc
+        class DatabaseError < StandardError; end
+
+        NativeException = java.lang.Exception
+             
+        # Default database error classes
+        DATABASE_ERROR_CLASSES = [NativeException].freeze
```

Comment:
Sorry, I'm new to jruby integration.

Doesn't this mean all native exceptions and all java.lang.Exceptions? That seems to cover a lot more than just the jdbc database exception classes.